bands/templates/bands/w9.html

```diff
@@ -10,6 +10,7 @@
 
 <form method="post" action="w9" enctype="multipart/form-data">
 <input type="hidden" name="id" value="{{ band.id }}" />
```

Comment:
That's doable, although with the agreement and bio page the existing data is loaded. Or... oh dear, is it making new BandBio objects each time?